app/policies/user_policy.rb

```diff
@@ -11,8 +11,12 @@ def create?
     is_admin? || is_himself?
   end
 
+  def update?
+    is_himself?
```

Comment:
We talk with Akita and others admin not be able to update another user, only himself.